crates/solidity/testing/snapshots/bindings_output/imports/default/generated/0.7.1.txt

```diff
@@ -0,0 +1,19 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+   ╭─[input.sol:1:1]
+   │
+ 4 │ function foo() returns (int) {
```

Comment:
I really like this new format!

I wonder if it is worth it to add a `success` or `failure` suffix to file names, to indicate the precence of any errors in the results?

Reply:
Hmm... I'm not sure. Because the tag would relate to parsing, not the bindings resolution. Maybe `parse-success`/`parse-failure` or is it too long?

Reply:
That's true. I'll add the suffix.